pull_request_template.md

```diff
@@ -0,0 +1,7 @@
+# PR Checklist
+
+- [ ] Do you have passing tests?
+- [ ] Did you run the linter?
+- [ ] Do you have tests? Tests passing?
+- [ ] Did you squash commits into 1 commit?
+- [ ] Do you have a Description showing changes made?
```

Comment:
Nice formatting
Items 1 and 3 seem like duplicates, is that correct?

Reply:
Also, just to get into the habit of things, this PR should also have a Description.